src/client/tile.h

```diff
@@ -244,6 +244,8 @@ struct TileLayer
 	{
 		switch (material_type) {
 		case TILE_MATERIAL_BASIC:
+			material.MaterialType = video::EMT_SOLID;
+			break;
```

Comment:
This case could be combined next to `case TILE_MATERIAL_LIQUID_OPAQUE:` a few lines below.